tests/test_model_evaluation.py

```diff
@@ -230,6 +233,24 @@ def sample_from_normal(rng):
     return within_bound
 
 
+@pytest.mark.parametrize("random_state", random_seeds)
+@pytest.mark.parametrize("test_size", [100, 300])
+def test_bootcross_split(random_state, test_size):
+    """Make sure the bootstrap - splitting is working as intended."""
+    N = 1000
+    random_state = check_random_state(random_state)
+    tri, tsi = _bootcross_split(N, test_size, random_state)
+    
+    # Test size of test set
+    assert len(tsi) == test_size
+
+    # Make sure training and testing are not overlapping
```

Comment:
checking emptiness of `set.intersection()` might be more efficient... insignificant in the scheme of things though

Reply:
Good point! I'll put this in for my next PR